### ISSUE TITLE
Use from command palette / keyboardshortcut

### ISSUE THREAD

**Issue by YayC** [reduckted/GitWebLinks#2]
When I select the `gitweblinks.copySelection` or `gitweblinks.copyFile` actions in the command palette (`shift+cmd+p`) nothing happens. I bound `gitweblinks.copySelection` to a keyboard shortcut and that doesn't work either. I'm on MacOS Catalina.

In the meantime this extension works great via right click, thanks for sharing it.

VSCode version:
```
Version: 1.42.0
Commit: ae08d5460b5a45169385ff3fd44208f431992451
Date: 2020-02-06T10:51:33.119Z
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.3.0
```

**Comment by YayC** [comment]
I actually use another extension (GitLens) that I realized has this functionality. I'll stick with that for now.

**Comment by reduckted** [comment]
Thanks for reporting this! 😁 

I initially designed the commands to be run from the context menus, but running from the command palette or via shortcut keys makes a lot of sense. The change was simple and I've published the changes in v1.5.0. That version should be available from the marketplace now.
